changelog/updates/2022-09-08-emerge-gitclone.md

```diff
@@ -0,0 +1 @@
+- emerge-gitclone: Migrate emerge-gitclone to use scripts repo tags and submodule refs
```

Comment:
I would put this under `changes/` or we can also leave it out since it's an internal change